hook.py

```diff
@@ -11,8 +12,10 @@ async def enable(services):
     app = services.get('app_svc').application
     debrief_gui = DebriefGui(services)
     app.router.add_static('/debrief', 'plugins/debrief/static/', append_version=True)
+    app.router.add_static('/logodebrief', 'plugins/debrief/uploads/', append_version=True)
     app.router.add_route('GET', '/plugin/debrief/gui', debrief_gui.splash)
     app.router.add_route('POST', '/plugin/debrief/report', debrief_gui.report)
     app.router.add_route('*', '/plugin/debrief/graph', debrief_gui.graph)
     app.router.add_route('POST', '/plugin/debrief/pdf', debrief_gui.download_pdf)
     app.router.add_route('POST', '/plugin/debrief/json', debrief_gui.download_json)
+    app.router.add_route('POST', '/plugin/debrief/uploadlogo', debrief_gui.upload_logo)
```

Comment:
here we should just make the end point /plugin/debrief/logo and then actions should be through the request type. that way we avoid later a 'DELETE' on /uploadlogo

Reply:
End point changed to `/plugin/debrief/logo`